projectgenerator/libili2pg/ilicache.py

```diff
@@ -130,29 +130,44 @@ def process_local_ili_folder(self, path):
         '''
         Parses all .ili files in the given ``path`` (non-recursively)
         '''
-        model = dict()
         models = list()
-        re_model = re.compile('\s*MODEL\s*([\w\d_-]+)\s.*')
-        re_model_version = re.compile('VERSION "([ \w\d\._-]+)".*')
-
+        fileModels = list()
         for ilifile in glob.iglob(os.path.join(path, '*.ili')):
-            with open(ilifile, 'r') as file:
-                for line in file:
-                    result = re_model.search(line)
-                    if result:
-                        model = dict()
-                        model['name'] = result.group(1)
-                        model['version'] = ''
-                        models += [model]
-
-                    result = re_model_version.search(line)
-                    if result:
-                        model['version'] = result.group(1)
-
-        print(repr(models[0]['name']))
+            try:
+                fileModels = self.parse_ili_file(ilifile, "utf-8")
```

Comment:
Yes, we can use `QMessageBar` to show a warning if latin1 is required and an error if another encoding is required.  We'll keep logging the error messages if you don't mind.